src/devices/legrand.ts

```diff
@@ -413,10 +413,11 @@ export const definitions: DefinitionWithExtend[] = [
         },
     },
     {
-        zigbeeModel: [" 3 Phases power consumption module\u0000\u0000", " DIN 3Ph power consumption module", "3ph Smart shedder module"],
+        zigbeeModel: [" 3P power consumption module\u0000\u0000\u0000", " DIN 3Ph power consumption module", " 3ph Smart shedder module"],
```

Comment:
```suggestion
        zigbeeModel: [" 3 Phases power consumption module\u0000\u0000", " 3P power consumption module\u0000\u0000\u0000", " DIN 3Ph power consumption module", "3ph Smart shedder module"],
```

To not make existing supported models unsupported.